arch/arm64/configs/eve_defconfig

```diff
@@ -555,6 +555,9 @@ CONFIG_HNS3=y
 CONFIG_HNS3_HCLGE=y
 CONFIG_HNS3_ENET=y
 CONFIG_NET_VENDOR_MEDIATEK=y
+CONFIG_NET_VENDOR_CADENCE=y
+CONFIG_MACB=y
```

Comment:
Please, after adding your changes to the eve_defconfig file, run `make eve_defconfig && make savedefconfig` and replace the `eve_defconfig` file with the generated `defconfig`, in this way we always keep only the needed set of options in the default config file and it's easier to merge the changes with other variants of our kernel as well...